src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -170,13 +172,15 @@ Every pull request needs these labels so we can see where our contributions come
 
 ## Docs issues [#docs-issues]
 
-Docs issues is a docs team program that enlists the help of product teams in solving documentation and product issues. The goal is to get product team expertise in solving issues filed by other relics and our users. Docs issues candidates are issues that would be difficult for a docs hero to chase down or would benefit from deep engineering expertise. Examples of good candidates are code examples, requirements sections, and NRQL queries.
+There's no hard and fast rule in choosing good candidates for docs issues. They could be anything too difficult for a docs hero to chase down, or anything that could benefit from deep engineering expertise. Some examples are code examples, requirements sections, and NRQL queries.
+
+Ultimately, docs issues helps the docs team collaborate with product teams to solve documentation and product issues. It lets us leverage product team expertise to solve as many issues as we can, regarldess of who filed them.
 
-The workflow for a viable issue looks something like this:
+The life cycle of a viable issues candidate looks something like this:
 
-1. A user files an GitHub issue and the GitHub hero identifies it as a potential docs issue.
-2. The hero marks the issue with the `docs-issue-candidate` label.
-3. Once a week, a designated docs team member goes through and assesses all issues with the `docs-issue-candidate` label.
-4. All approved issues get marked as `docs-issue` and migrated to the customer feedback Jira project.
-5. The new jiras are eventually triaged by PMs and funneled to the correct project team.
-6. The assignee works on the jira/issue and closes them once complete.
+1. Someone files an issue through GitHub for the GitHub hero to evaluate it as a potential candidate. 
+2. The hero marks it using the `docs-issue-candidate` label.
+3. Every week, another designated docs team member assesses all issues with the `docs-issue-candidate` label.
+4. Once approved, that same designated docs team member will change the label from `docs-issue-candidate` to `docs-issue`. We then migrate the issue into the customer feedback Jira project.
```

Comment:
That sentence feels a bit contorted.... How about something like: "If approved, the label is changed to `docs-issue`..."